mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cfg/HibernateOrmMapperOutboxPollingSettings.java

```diff
@@ -511,6 +523,7 @@ private CoordinationRadicals() {
 		public static final String ENTITY_MAPPING_OUTBOXEVENT_SCHEMA = ENTITY_MAPPING_OUTBOXEVENT_PREFIX + "schema";
 		public static final String ENTITY_MAPPING_OUTBOXEVENT_CATALOG = ENTITY_MAPPING_OUTBOXEVENT_PREFIX + "catalog";
 		public static final String ENTITY_MAPPING_OUTBOXEVENT_UUID_GEN_STRATEGY = ENTITY_MAPPING_OUTBOXEVENT_PREFIX + "uuid_gen_strategy";
+		public static final String ENTITY_MAPPING_OUTBOXEVENT_UUID_DATA_TYPE = ENTITY_MAPPING_OUTBOXEVENT_PREFIX + "preferred_uuid_jdbc_type";
```

Comment:
As far as I can tell this is not just a preference, but a mandated type which Hibernate Search will always comply with.

Would you mind renaming `preferred_uuid_jdbc_type` to `uuid_jdbc_type`?

Reply:
Sure, will change; I've used the key as in ORM so that it might look more familiar to the users but can see how the "preferred" part is a bit confusing 😄